charts/k8s-monitoring/templates/alloy_config/_metrics_service_remote_write.alloy.txt

```diff
@@ -12,6 +12,12 @@ prometheus.remote_write "metrics_service" {
       username = nonsensitive(remote.kubernetes.secret.metrics_service.data[{{ .basicAuth.usernameKey | quote }}])
       password = remote.kubernetes.secret.metrics_service.data[{{ .basicAuth.passwordKey | quote }}]
     }
+{{- else if eq .authMode "bearerToken" }}
+  {{- if .bearerToken.token }}
+    bearer_token = remote.kubernetes.secret.metrics_service.data[{{ .bearerToken.tokenKey | quote }}]
+  {{- else if .bearerToken.tokenFile }}
+    bearer_token_file = {{  .bearerToken.tokenFile | quote  }}
+  {{- end }}
```

Comment:
```suggestion
  {{-  if .bearerToken.tokenFile }}
    bearer_token_file = {{  .bearerToken.tokenFile | quote  }}
  {{- else }}
    bearer_token = remote.kubernetes.secret.metrics_service.data[{{ .bearerToken.tokenKey | quote }}]
  {{- end }}
```

If a user uses an external (pre-created) secret, the bearerToken wont be in the values, but we still want to fetch it from the secret. So, that also means we should check if they passed the file first.

Reply:
added changes as directed